include/kwk/detail/sequence/prefilled.hpp

```diff
@@ -43,7 +43,7 @@ namespace kwk::__
     using value_type                            = typename descriptor_t::base_type;
     static constexpr  std::int32_t storage_size = 0;
 
-    struct storage_type {};
+    struct storage_type { constexpr auto operator==(storage_type const&) const noexcept { return true; } };
```

Comment:
oh why is it needed ?

Reply:
for the fast-path prefilled comparison operator :)